tests/test_audio_data.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import subprocess
+import shutil
+
+from pathlib import Path
+from dragtor.audio_loader import AudioLoader
+from dragtor import config
+
+project_root = Path(__file__).parent.parent
+local_audio_sample = project_root / "tests" / "assets" / "audio_sample" / "sample.wav"
+local_invalid_audio_sample = project_root / "tests" / "assets" / "audio_sample" / "sample99.mp3"
```

Comment:
you can have a look in conftest.py how you can leverage importlib.resources for file handling in tests